assets/js/components/Sparkline.js

```diff
@@ -66,6 +66,11 @@ function Sparkline( {
 		],
 	};
 
+	if ( ! data.slice( 1 ).some( ( row ) => row[ 1 ] > 0 ) ) {
+		chartOptions.vAxis.minValue = 0;
+		chartOptions.vAxis.maxValue = 1;
+	}
```

Comment:
@johnPhillips could you please add a comment to explain why we need this?

Reply:
Of course, sorry 👍